packages/lyve_cloud/kibana/dashboard/lyve_cloud-42d12a00-5852-11ed-a702-0d7be6ba8b3f.json

```
@@ -0,0 +1,223 @@
+{
+    "attributes": {
+        "description": "Shows the total S3 API calls for the service accounts, separated to buckets.",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": {
+                "filter": [],
+                "query": {
+                    "language": "kuery",
+                    "query": ""
+                }
+            }
```

```suggestion
            "searchSourceJSON": {
                "filter": [],
                "highlightAll": true,
                "query": {
                    "language": "kuery",
                    "query": "data_stream.dataset:lyve_cloud.audit"
                },
                "version": true
            }
```

Is better if we embed the query filter with the dashboard